src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTextFixture.kt

```diff
@@ -109,6 +112,8 @@ open class CodyIntegrationTextFixture : BasePlatformTestCase(), LensListener {
   }
 
   private fun checkInitialConditions() {
+    isAuthenticated()
```

Comment:
It does not matter.

---

With the check the tests fail with `User is not authenticated` assertion error. 

Without it it fails with:
```
junit.framework.ComparisonFailure: Action description should be empty expected:<[]> but was:<[Log in to Sourcegraph to enable Cody features]>
	at junit.framework.Assert.assertEquals(Assert.java:100)
	at junit.framework.TestCase.assertEquals(TestCase.java:253)
	at com.sourcegraph.cody.util.CodyIntegrationTextFixture.checkInitialConditions(CodyIntegrationTextFixture.kt:140)
```